### ISSUE TITLE
`aio templates install` needs a flag so the install.yml is not processed

### ISSUE THREAD

**Issue by shazron** [adobe/aio-cli-plugin-app-templates#13]
For PR https://github.com/adobe/aio-cli-plugin-app-templates/pull/6

For `aio app init --no-login`, a user can install templates without being logged in. The app plugin uses the `templates` plugin to install the templates, so it needs to pass a flag to tell the `templates:install` plugin not to process the template's `install.yml` file.

e.g.
`aio templates:install @adobe/generator-app-excshell --no-process-install-config`

Basically a flag `process-install-config` with default of `true` and `allowNo: true`. Only process the `install.yml` if `flags["process-install-config"]` is `true`.

**Comment by shazron** [comment]
Tracked internally at https://jira.corp.adobe.com/browse/DEVX-2367

**Comment by irenelagno** [comment]
@sarahxxu FYI

**Comment by shazron** [comment]
Completed in beta.5
